yocs_joyop/launch/joyop.launch

```diff
@@ -5,33 +5,41 @@
  -->
 
 <launch>
-  <arg name="linear_axis"         default="1"/>
-	<arg name="angular_axis"        default="0"/>
-	<arg name="linear_scale"        default="1.0"/>
-	<arg name="angular_scale"       default="1.0"/>
-	<arg name="deadman_button"      default="4"/>
-	<arg name="enable_button"       default="0"/>
-	<arg name="disable_button"      default="1"/>
-	<arg name="spin_frequency"      default="10"/>
-	<arg name="joy_topic_name"      default="joy"/>
-	<arg name="cmd_vel_topic_name"  default="cmd_vel"/>
-	<arg name="enable_topic_name"   default="enable"/>
-	<arg name="disable_topic_name"  default="disable"/>
+    <arg name="linear_axis"           default="1"/>
```

Comment:
There were tabs in it before, so I just let eclipse roll over it with its default formatting for xml files which is four spaces. Didn't know there was a ros convention, let's use that.